test/jest.matchers.ts

```diff
@@ -0,0 +1,130 @@
+import { BigNumber, BigNumberish } from "ethers";
+
+expect.extend({
+  toBnEquals: (b1: BigNumberish, b2: BigNumberish) => {
```

Comment:
I would be ok to shorten these matchers names so it uses less code space (but not mandatory)
something like `toBnBe` or `toBnEq`, `toBnGte`